src/FileStream.cpp

```diff
@@ -55,14 +56,14 @@ bool pocolog_cpp::FileStream::open(const char* fileName, std::ios_base::openmode
     writePos = 0;
     this->fileName = fileName;
     
-//     std::cout << "File opened " << (fd > 0) << " file size " << fileSize  << " blk size " << blockSize << std::endl;
+    LOG_DEBUG_S << "File opened " << (fd > 0) << " file size " << fileSize  << " blk size " << blockSize << std::endl;
```

Comment:
LOG_DEBUG_S adds a std::endl, so you could remove it here.

Reply:
All std::endls are removed, except from executables.

Reply:
All std::endls are removed now.